src/ngraph/runtime/cpu/unit_test.manifest

```diff
@@ -4,9 +4,9 @@ shape_of_matrix
 shape_of_5d
 sum_stable_acc_double
 
-# failing in CI build but passing on local machine
+#failing in CI build but passing on local machine
 max_3d_to_scalar_int32
 
-# Not implemented
-erf
-zero_sized_erf
+#Not implemented
+erf zero_sized_erf
```

Comment:
I think this `erf zero_sized_erf` should be split into 2 lines:
```
erf
zero_sized_erf
```

Reply:
Good catch, fixed